schemas/checks/check.yaml

```diff
@@ -47,6 +47,9 @@
       applicant_id:
         type: string
         description: The ID of the applicant to do the check on.
+      privacy_notices_read_consent_given:
```

Comment:
I think this needs `writeOnly: true`, as in API v3 we're only adding to the request object

Reply:
Done!